### PR TITLE
.*: support HA

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

support ha

### What is changed and how it works?

merge code from ha-dev, and refine some import's format

### Check List 

Side effects
 - Breaking backward compatibility

Related changes

 - Need to update the documentation
 - Need to update the `dm/dm-ansible`

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@lichunzhu @csuzhangxc PTAL

File: dm/master/config.go [hunk 1]
```diff
@@ -50,13 +50,15 @@ var SampleConfigFile string
 // NewConfig creates a config for dm-master
 func NewConfig() *Config {
 	cfg := &Config{}
+	cfg.Debug = false
```
Comment (lichunzhu) [inline dm/master/config.go]: It sames that `cfg.Debug` is useless now. We can delete it in next PR.

Comment (csuzhangxc) [reply dm/master/config.go]: some more useless code needs to be deleted later.

**Pull request merged**